monitors/onlinemon/src/CorrelationCollection.cc

```diff
@@ -357,7 +370,11 @@ void CorrelationCollection::fillHistograms(const SimpleStandardPlane &p1,
 
   std::pair<SimpleStandardPlane, SimpleStandardPlane> plane(p1, p2);
   CorrelationHistos *corrmap = _map[plane];
-  if (corrmap) {
```

Comment:
this change also should not be required?

Reply:
I think there is `if(corrmap == NULL)` in the next line which does the same thing